### DISCUSSION TITLE
Twitter-like threads

### DISCUSSION THREAD

**Discussion by chetanyakan** [mattermost/community#121] [Moonshot Ideas]
Could be wrapped under a system console setting.

Each post in the thread can have one or more children.
Starting few posts from the thread author are clubbed together.

**Comment by Willyfrog** [comment]
I had a similar idea with the custom statuses https://github.com/mattermost/community/discussions/54

**Comment by esethna** [comment]
Threads on threads can get really messy from a UX perspective, but I appreciate the pain point behind this. I would suggest pop-out threads so you can just create a new thread from any reply in the existing thread
